yjit.c

```diff
@@ -716,6 +716,14 @@ rb_get_iseq_body_param_opt_table(const rb_iseq_t *iseq)
     return iseq->body->param.opt_table;
 }
 
+VALUE
+rb_optimize_call(VALUE *recv, rb_execution_context_t *ec, int argc, VALUE *argv, int kw_splat, VALUE block_handler) {
```

Comment:
nitpick: It seems a bit inconsistent with other functions

```suggestion
rb_optimize_call(VALUE *recv, rb_execution_context_t *ec, int argc, VALUE *argv, int kw_splat, VALUE block_handler)
{
```